scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1012,6 +1083,8 @@ public boolean resumeJob(JobId jobId) throws NotConnectedException, UnknownJobEx
      */
     @Override
     public boolean killJob(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        blockIfJobIsStillUnderSubmission(jobId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)